test/lib/mayaUsd/render/mayaToHydra/testPolygonPrimitives.py

```diff
@@ -0,0 +1,360 @@
+# Copyright 2024 Autodesk
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import maya.cmds as cmds
+import fixturesUtils
+import mtohUtils
+
+import platform
+
+class TestPolygonPrimitives(mtohUtils.MtohTestCase):
+    # MayaHydraBaseTestCase.setUpClass requirement.
+    _file = __file__
+
+    @property
+    def imageDiffFailThreshold(self):
+        return 0.1
+    
+    @property
+    def imageDiffFailPercent(self):
+        if platform.system() == "Darwin":
+            return 2
```

Comment:
Ouch.  Would be nice to have a JIRA ticket number in a comment, so that we can exhaustively find and remove these once the underlying cause is found and fixed.

Reply:
I've added it here and in all other places where we have such a workaround